admin/app/helpers/solidus_admin/container_helper.rb

```diff
@@ -9,7 +9,7 @@ def container
     end
 
     def component(name)
-      container.resolve("#{name}_component")
+      container.resolve("components.#{name}.component")
```

Comment:
Does it make sense to change the keys to use `/` instead of `.` in the context of this PR or it's better to use a different one?

Reply:
Thanks for your feedback. Let's talk about it in a different place. I'm a bit reticent, as the identifier doesn't need to map one-to-one to a filesystem path, so it could lead to confusion :thinking: Actually, that's already the case, as `components.main_nav.component` maps to `components/solidus_admin/main_nav/component.rb`.